### PR TITLE
Allow for LCP download links in side panel

### PR DESCRIPTION
**Pull request by iisa**
- add link option for LCP in downloads section
- add helpful text section to download SimplyE, Aldiko Next, Thorium
- add in IA demo a toggle to show LCP download links

To test: https://deploy-preview-1061--lucid-poitras-9a1249.netlify.app/bookreaderdemo/demo-internetarchive?ocaid=theworksofplato01platiala
- once review app is loaded, open side menu, then click "LCP Download Option" -> options should update with the helper text showing

image

### CODE REVIEW

File: src/BookNavigator/downloads/downloads.js [hunk 1]
```diff
@@ -40,6 +40,16 @@ export class IABookDownloads extends LitElement {
     ));
   }
 
+  /**
+   * checks if downloads list contains an LCP option
+   * @return {boolean}
+   */
+  get hasLCPOption() {
+    const regex = /^(LCP)/g;
+    const lcpAvailable = this.downloads.some(option => option.type?.match(regex));
```
Comment (nsharma123) [inline src/BookNavigator/downloads/downloads.js:49]: So you are assuming that LCP will be available in both version always. Any change LCP available for PDF version only?

File: src/BookNavigator/downloads/downloads.js [hunk 2]
```diff
@@ -59,12 +69,24 @@ export class IABookDownloads extends LitElement {
     `;
   }
 
+  get installSimplyEAldikoThoriumMsg() {
+    return html`
+    <p>For LCP downloads, make sure you have SimplyE or Aldiko Next installed on mobile or Thorium on desktop.</p>
+    <ul>
+      <li><a href="https://librarysimplified.org/simplye/" rel="noopener noreferrer" target="_blank">Install SimplyE</a></li>
+      <li><a href="https://www.demarque.com/en-aldiko" rel="noopener noreferrer" target="_blank">Install Aldiko</a></li>
+      <li><a href="https://www.edrlab.org/software/thorium-reader/" rel="noopener noreferrer" target="_blank">Install Thorium</a></li>
```
Comment (nsharma123) [inline src/BookNavigator/downloads/downloads.js:78]: Did you thought about `nofollow` as well inside rel attribute?

File: src/BookNavigator/downloads/downloads.js [hunk 3]
```diff
@@ -59,12 +69,24 @@ export class IABookDownloads extends LitElement {
...
+  }
+
   render() {
     return html`
       ${this.header}
       ${this.loanExpiryMessage}
       <ul>${this.renderDownloadOptions()}</ul>
-      ${this.isBookProtected ? this.accessProtectedBook : nothing}
+      ${this.hasLCPOption ? this.installSimplyEAldikoThoriumMsg : nothing}
+      ${this.isBookProtected && !this.hasLCPOption ? this.accessProtectedBook : nothing}
```
Comment (bfalling) [inline src/BookNavigator/downloads/downloads.js:89]: TINY SUGGESTION: I think you could slightly simplify the logic by nesting the conditions, e.g.
```
${this.hasLCPOption
  ? this.installSimplyEAldikoThoriumMsg
  : (this.isBookProtected ? this.accessProtectedBook : nothing)
}
```
Not sure it's an improvement, but see what you think! ;)

**Review by bfalling — APPROVED** [review]
One tiny suggestion. Otherwise, LGTM!

File: src/BookNavigator/downloads/downloads.js (hunk 2 above)
Comment (iisa) [reply src/BookNavigator/downloads/downloads.js:78]: thanks for the catch. updated.

File: src/BookNavigator/downloads/downloads.js (hunk 1 above)
Comment (iisa) [reply src/BookNavigator/downloads/downloads.js:49]: > Any change LCP available for PDF or EPUB version only? 

unsure, but they can use any of the new option types to add a link so we should be covered. `option.type: 'LCP ePub' | 'LCP PDF'`

File: src/BookNavigator/downloads/downloads.js (hunk 3 above)
Comment (iisa) [reply src/BookNavigator/downloads/downloads.js]: hi, very good suggestion! updated.

**Pull request merged**